### PR TITLE
Fix/negative rating

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

**Comment by Segelzwerg** [comment]
Probably a bug in actions https://github.com/Whist-Team/actions/issues/4

**Comment by Segelzwerg** [comment]
> Probably a bug in actions https://github.com/Whist-Team/actions/issues/4

Or user error 😉

File: whist_core/user/player.py [hunk 1]
```diff
@@ -22,6 +25,17 @@ def __eq__(self, other):
             return False
         return other.username == self.username
 
+    # Pydantic will convert this into a classmethod, cls is the correct parameter
+    @validator('rating')
+    # noinspection ImproperFirstParameter
+    def rating_must_not_be_negative(cls, value):  # pylint: disable=no-self-argument
+        """
+        Validates the rating. It must be zero or positiv.
```
Comment (iTitus) [inline whist_core/user/player.py:33]:
```suggestion
        Validates the rating. It must be zero or positive.
```

File: whist_core/error/player_error.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+"""Exceptions that occur in player or user"""
+
+
+class NegativeRatingError(Exception):
+    """
+    Will be raised if a attempt to assign a negative rating to a player"""
```
Comment (iTitus) [inline whist_core/error/player_error.py:6]: Please fix the grammar.

**Review by iTitus — CHANGES_REQUESTED** [review]

**Review by iTitus — APPROVED** [review]

**Pull request merged**